src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import React, { useState } from 'react';
+import { findById, addId } from './helpers';
+import { User, Todos } from './types/types';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const todos = todosFromServer.map(todo => {
+  const user = usersFromServer.find(
+    findById<User>(todo.userId),
+  ) || null;
```

Comment:
https://github.com/mate-academy/react_add-todo-form/pull/2221#discussion_r1418579045 - I have already offered a ready-made option for using the `find`, it remains to make it right as a helper